core-primitives/node-api/api-client-types/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "itp-api-client-types"
-version = "0.1.0"
-edition = "2021"
+version = "0.8.0"
+edition = "2018"
```

Comment:
Compared to our other crates, the `authors` is still missing. Do we not want consistency in that too?

Reply:
Hahah, thanks for pointing this out too. Yes, we should stay consistent, and we should also test this somehow with a script in the integration tests, otherwise we will eventually get inconsistent.